src/snowflake/connector/cursor.py

```diff
@@ -974,8 +974,7 @@ def execute(
                 )
             query = query1
 
-        if logger.getEffectiveLevel() <= logging.INFO:
```

Comment:
Do you know the purpose of this? Looks not useful.

Reply:
It's not useful anymore